examples/example-hono-api/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "example-hono-api",
+  "version": "1.0.0",
+  "description": "Hono API example for auth0-api-js deployable as Cloudflare worker",
+  "type": "module",
+  "scripts": {
+    "deploy": "wrangler deploy",
+    "dev": "wrangler dev",
+    "cf-typegen": "wrangler types"
+  },
+  "devDependencies": {
+    "@auth0/auth0-api-js": "*",
+    "@cloudflare/workers-types": "^4.20250321.0",
+    "hono": "^4.7.5"
```

Comment:
Is this expected as a devDependencies? I assume this is fine because of how hono works, but just checking.